apps/web/src/pages/activities/ActivitiesPage.tsx

```diff
@@ -21,7 +21,7 @@ interface IFiltersForm {
 }
 
 export function ActivitiesPage() {
-  const { templates, loading: loadingTemplates } = useTemplates();
+  const { templates, loading: loadingTemplates } = useTemplates(0, false);
```

Comment:
Here instead of passing the usePagiatnion, we can just pass the maximum value 100, and in the future just do a server side text search to fix the large scale issues here.